### ISSUE TITLE
Add Osiris settings to rabbitmq.conf

### ISSUE THREAD

**Issue by acogoluegnes** [rabbitmq/rabbitmq-server#4014]

**Comment by michaelklishin** [comment]
Note that for Ra settings we use the `raft` prefix, not `ra`. So maybe here it would make sense to use `streams` or something similar, as `osiris` won't make much sense to most users.

**Comment by acogoluegnes** [comment]
Good point. We already use the `stream` prefix for the [stream plugin](https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbitmq_stream/priv/schema/rabbitmq_stream.schema) (no `s`), this is the usual convention for plugins. I understand that `streams` makes more sense than `osiris`, but I'm afraid `stream` and `streams` could confuse people. What's your take on this?
